frontend/src/screens/TemplatesList/TemplateCard.tsx

```diff
@@ -0,0 +1,70 @@
+import { Button } from 'semantic-ui-react';
+import { colors } from '../../utils/colors';
+import { Template, TemplateFolder } from '../../utils/types';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faTrash } from '@fortawesome/free-solid-svg-icons';
+import { subtitleFontSize } from '../../utils/sizes';
+
+interface TemplateCardProps {
+  template: Template;
+  handleDelete: () => void;
+}
+
+const TemplateCard = ({ template, handleDelete }: TemplateCardProps) => {
+  const getFolderTreeElememt = (structure: TemplateFolder[]): React.ReactElement | null => {
+    const createTree = (node: TemplateFolder): React.ReactElement | null => {
+      const children = structure.filter((child) => child.parent === node.folder);
+      if (children.length === 0) {
+        return <div key={node.folder}>📁 {node.folder}</div>;
+      }
+
+      return (
+        <div key={node.folder}>
+          📁 {node.folder}
+          <div style={{ marginLeft: '20px' }}>{children.map((child) => createTree(child))}</div>
+        </div>
+      );

```

Comment:
Semantic provides lists that could be useful here.

Reply:
I will stick to custom list